### ISSUE TITLE
Documentation mentions FeatureValueToId, but it's not found anywhere on the site.

### ISSUE THREAD

**Issue by smsorin** [tensorflow/tensorflow#13519]
The latest documentation on tensorflow.org expains that for embedding_lookup* functions ids are obtained typically from FeatureValueTold. The last term is not found anywhere else on the site. Probably some remnant from ancient functions.

It should probably be replaced with index_table_from* or something similar.

**Comment by cy89** [comment]
@smsorin could you add a link to the bad documentation, please? I'm not seeing "FeatureValueTold" on tensorflow.org, but it could be that I'm not searching well enough.

**Comment by smsorin** [comment]
https://www.tensorflow.org/versions/master/api_docs/python/tf/nn/embedding_lookup_sparse

sp_ids: N x M SparseTensor of int64 ids (typically from FeatureValueToId), where N is typically batch size and M is arbitrary.

I was seeing it on tf.nn.embedding_lookup as well but that seems to be fixed now.

**Comment by cy89** [comment]
I think we got bit by the similarity of the glyphs for capital-i, and lowercase-L. "FeatureValueToId" gets a few more hits, while "FeatureValueTold" isn't on the site. 

However, even FeatureValueToId doesn't have much visible code or documentation. @zheng-xq, are the docs out of date (so we should remove the old reference and replace it with something more recent)?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Awaiting TensorFlower: It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
A member of the TensorFlow organization has replied after the stat:awaiting tensorflower label was applied.

**Comment by tatatodd** [comment]
@ysuematsu might have some ideas.

**Comment by tensorflowbutler** [comment]
Nagging Awaiting TensorFlower: It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Awaiting TensorFlower: It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Awaiting TensorFlower: It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging TensorFlower: It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by poxvoculi** [comment]
opened internal bug.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @alexgorban: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @alexgorban: It has been 16 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @alexgorban: It has been 47 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @alexgorban: It has been 62 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @alexgorban: It has been 77 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by poxvoculi** [comment]
This should have been fixed around the end of April.
